services/dqa-secondary-reports/report_parser.py

```diff
@@ -81,11 +84,21 @@ def parse(self):
                 setdefault(code, {}).setdefault(status, {}).\
                 setdefault(self.hospital, []).append(field_record)
 
+            self.site_table_totals.setdefault(self.table, {}).\
```

Comment:
This is up to you, but this type of boilerplate could be simplified greatly by using `collections.defaultdict`

```python
from collections import defaultdict

# Recursive nested dict structure.
rdict = lambda: defaultdict(rd)

self.site_table_totals = rdict()
self.site_table_totals[self.table][self.code][self.status] = []
```